vspace/src/main/java/edu/asu/diging/vspace/core/services/IImageService.java

```diff
@@ -37,4 +37,6 @@ public interface IImageService {
 
     List<IVSImage> findByFilenameOrNameContains(String searchTerm);
 
+    IVSImage getImageByDescription(String imageId) throws ImageDoesNotExistException;
```

Comment:
why is this method called `getImageByDescription` if it is given an image id (and not a description)?